website/docs/core/applications.md

```diff
@@ -54,3 +54,13 @@ Requires authentik 2022.3
 :::
 
 To give users direct links to applications, you can now use an URL like `https://authentik.company/application/launch/<slug>/`. This will redirect the user directly if they're already logged in, and otherwise authenticate the user, and then forward them.
+
+### Backchannel providers
+
+:::info
+Requires authentik 2023.5
+:::
+
+Backchannel providers can augment the functionality of applications with additional protocols. The main provider of an application provides the SSO protocol used for logging into the application. Backchannel providers can be used for protocols such as [SCIM](../providers/scim/index.md) and [LDAP](../providers/ldap/index.md) to provide directory syncing.
```

Comment:
```suggestion
Backchannel providers can augment the functionality of applications by using additional protocols. The main provider of an application provides the SSO protocol that is used for logging into the application. Then, an additional backchannel provider can be used for protocols such as [SCIM](../providers/scim/index.md) and [LDAP](../providers/ldap/index.md) to provide directory syncing.
```

Reply:
very slight nit, an application can have multiple backend providers (even of the same kind) hence I used the plural